addons/openebs/addon.rb

```diff
@@ -43,7 +43,7 @@
 
   reset_host { |host|
     data_dir = config.default_storage_pool[:path].strip
-    host.ssh.exec("sudo rm -rf #{data_dir}/*") unless data_dir.empty?
+    host.exec("sudo rm -rf #{data_dir}/*") unless data_dir.empty?
```

Comment:
These could actually be `transport.exec` and the host wouldn't need the delegator.